### PR TITLE
20 favourites location dropdown

### PR DESCRIPTION
**Pull request by ekauf**
pls review

### CODE REVIEW

**Review by DanForder — CHANGES_REQUESTED** [review]
fix merge conflicts

File: src/components/DropDown/DropDown.scss [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  }
+}
+
+.arrow-active {
+  transform: rotate(180deg);
+}
+
+.dropdown__options {
+  position: absolute;
+}
```
Comment (DanForder) [inline src/components/DropDown/DropDown.scss:49]: this can be nested inside the `.dropdown` class

File: src/components/DropDown/DropDown.scss [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    .dropdown__image-container {
+      width: 20%;
+    }
+  }
+}
+
+.arrow-active {
+  transform: rotate(180deg);
+}
```
Comment (DanForder) [inline src/components/DropDown/DropDown.scss:45]: should this be `dropdown__arrow--active` like in the jsx?

**Review by DanForder — CHANGES_REQUESTED** [review]

File: src/components/DropDown/DropDown.scss (hunk 2 above)
Comment (ekauf) [reply src/components/DropDown/DropDown.scss:45]: I dont think so. In the jsx it is: dropdown__arrow arrow-active

Comment (DanForder) [reply src/components/DropDown/DropDown.scss:49]: either way, any classes in this file should start with `dropdown__`

Comment (ekauf) [reply src/components/DropDown/DropDown.scss]: has been resolved, thank you for the clarification

**Review by DanForder — CHANGES_REQUESTED** [review]
fix linting errors, remove unused variables

**Review by DanForder — APPROVED** [review]

**Review by GeorgeSwift16 — APPROVED** [review]

**Pull request merged**